### PR TITLE
Changes the laser projectile icon. Adds a new one when it hits a wall.

### PR DESCRIPTION
**Pull request by lzimann**
:cl: Lexorion
tweak: Laser projectiles have a new sprite! They also have a new effect when they hit a wall.
/:cl:

Sprites by Lexorion:

4f5800924086d68600315f2fad9e8378

Got too lazy to convert these to gif so check them: 
[New heavy laser](https://file.house/3PU3.mp4)

[New regular laser](https://file.house/6Kpa.mp4)

### CODE REVIEW

**Comment by Kromgar** [comment]
HOLY SHIT SO PRETTY

**Comment by GunHog** [comment]
Sprite upgrades!!

**Comment by OtherDalfite** [comment]
Cool! I really like the fact that they glow now. Whats the orange beam for?

**Comment by OtherDalfite** [comment]
Nvm im dumb

File: code/game/objects/effects/overlays.dm [hunk 1]
```diff
@@ -551,6 +551,9 @@
 	icon_state = "impact_laser"
 	duration = 4
 
+/obj/effect/overlay/temp/impact_effect/red_laser/wall
+	icon_state = "impact_laser_wall"
```
Comment (ChangelingRain) [inline code/game/objects/effects/overlays.dm]: set duration to 10

**Comment by dannno** [comment]
cool

**Comment by Thunder12345** [comment]
http://puu.sh/tI2xP/616460f4c3.dmi

Recoloured the new sprites to replace every other beam

**Comment by XDTM** [comment]
Is there a way to greyscale it so i can make a rainbow laser gun?

**Comment by Thunder12345** [comment]
See my DMI

**Comment by AnturK** [comment]
Conflicts

**Pull request merged**